upstream-community-operators/infinispan/infinispan-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,181 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: infinispan-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    tectonic-visibility: ocs
+    capabilities: Basic Install
+    categories: "Database"
+    alm-examples: |
+      [
+        {
+          "apiVersion": "infinispan.org/v1",
+          "kind": "Infinispan",
+          "metadata": {
+            "name": "example-infinispan"
+          },
+          "spec": {
+            "size": 3
+          }
+        },
+        {
+          "apiVersion": "infinispan.org/v1",
+          "kind": "Infinispan",
+          "metadata": {
+            "name": "example-infinispan"
+          },
+          "config": {
+            "sourceType": "ConfigMap",
+            "sourceRef": "infinispan-app-configuration",
+            "name": "cloud-ephemeral.xml"
+          },
+          "spec": {
+            "size": 3
+          }
+        }
+      ]
+    description: TBD
+    containerImage: docker.io/jboss/infinispan-operator:0.1.0
+    createdAt: 2019-02-28T14:18:00Z
+    support: Infinispan
+    certified: "false"
+spec:
+  displayName: infinispan
+  description: |
+    TBD
+
+    TBD
+
+    TBD
+  keywords: ['infinispan', 'key value', 'database', 'datagrid', 'open source']
+  version: 0.1.0
+  maturity: stable
+  replaces: infinispan-operator.v0.0.0
+  maintainers:
+    - name: Infinispan Community
+      email: infinispan-dev@lists.jboss.org
+  provider:
+    name: Red Hat
+  labels:
+    operated-by: infinispan-operator
+  selector:
+    matchLabels:
+      operated-by: infinispan-operator
+  links:
+    - name: Blog
+      url: https://blog.infinispan.org/
+    - name: Documentation
+      url: TBD
+    - name: etcd Operator Source Code
+      url: https://github.com/infinispan/infinispan-operator
+
+  icon:
+    - base64data: TBD
```

Comment:
Please omit the icon if none is present. This might break the UI

Reply:
Commented for the time being. I don't want us to forget about it.